certify-service/src/main/java/io/mosip/certify/services/VCIPluginSelecter.java

```diff
@@ -0,0 +1,7 @@
+package io.mosip.certify.services;
+
+import io.mosip.certify.api.dto.VCRequestDto;
+
+public interface VCIPluginSelecter {
```

Comment:
this can be handled by a factory right, rather than having an interface and implementation